src/components/Menu/Menu.module.css

```diff
@@ -28,8 +28,20 @@
   font-weight: var(--eds-font-weight-light);
 }
 
-/* Needed to create higher specificity than ClickableStyle. */
-.menu__button--icon {
+.menu__plain-button {
```

Comment:
would be good to add a comment that this is for accessibility. note for the future: might be useful to extract this pattern

Reply:
Added!